packages/core/src/components/textarea/test/default/textarea.e2e.ts

```diff
@@ -0,0 +1,37 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/textarea/test/default/index.html';
+
+test.describe('tds-textarea', () => {
+  test('renders default textarea correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const textarea = page.locator('tds-textarea');
+    await expect(textarea).toHaveClass(/hydrated/);
```

Comment:
Just a wonder, should we really test this? That the class `hydrated` is present?

Reply:
Looks like the reviewers where ok with it in #374 
I just took that test from there and I guess if hydrated would somehow get removed as class is no bueno and the test should catch that?

Reply:
Ping @alexandereneroth @theJohnnyMe @nathalielindqvist @EmelieLitwin

Reply:
Oh I didn't know that was stencil stuff :) Sorry im so new. yeah then we shouldn't test it according to the guidelines and stencil should do those tests.

Reply:
No worries man, we are also new :)! I think Martin added it additionally, and then we briefly touched upon it in a meeting the other week. So like I said, from my perspective it feels weird to test for it. But, if we in the team feels it provides value then we can do it.

Reply:
Ok, I removed it for now and we can add those tests again later I think. This is not a final PR for adding tests, it only handles some of the tests that we have in the [unit test spec sheet](https://scaniaazureservices-my.sharepoint.com/:x:/r/personal/tatyana_bumazhkina_scania_com/Documents/Unit%20test%20scenarios.xlsx?d=wca48bf6d92cf4e8c8f456337e2002ec4&csf=1&web=1&e=9iV53q&nav=MTVfezdBNzNBRjE3LTNBNjUtNDMyQy1BRkZFLTU0NkQ4OEE0QTRBRX0)

Reply:
Yeah this came up on the testing start-up meeting as well. I think we said we don't need to test it since it's Stencil specific like you said @timrombergjakobsson, nice catch! We can probably remove it from the tests for now and we can add it back in if we see that it provides value later on :) great work so far @alfredskeden! 👏🎉

Reply:
Yes, I think it is better that we focus on verifying the actual behavior and functionality of our components, such as interactions, event handling, rendering correctness, and expected output etc. We have to trust Stencil.js and their internal testing of their own "stuff" :). But awesome work @alfredskeden! Fun stuff happening!!